contracts/cw-ics721-bridge/Cargo.toml

```diff
@@ -22,11 +22,12 @@ cw721 = { workspace = true }
 cw721-base = { workspace = true, features = ["library"] }
 thiserror = { workspace = true }
 serde = { workspace = true }
-cw-paginate = { workspace = true }
+cw-paginate-storage = { workspace = true }
 cw721-proxy-derive = { workspace = true }
 cw-pause-once = { workspace = true }
 cw-cii = { workspace = true }
 zip-optional = { workspace = true }
+cw721-init-msg-registry = { workspace = true, features = ["ics721-base", "cw721-base"], default-features = false}
```

Comment:
So for a custom ics721 contract like `sg-ics721-bridge` for Stargaze we need to fork repo and adjust this entry for using `sg721-base`?